Dockerfile

```diff
@@ -9,11 +9,10 @@ RUN apk add --no-cache \
       conntrack-tools \
       curl \
       bash && \
-    mkdir -p /var/lib/gobgp && \
-    mkdir -p /usr/local/share/bash-completion && \
-    curl -L -o /usr/local/share/bash-completion/bash-completion \
```

Comment:
@murali-reddy is there a runtime need for this bash-completion or would it be safe to remove and minimize?